rdt/transformers/pii/anonymizer.py

```diff
@@ -199,6 +199,10 @@ def _function(self):
 
         return result
 
+    def _fallback_function(self):
+        """Return the result of calling a fallback ``faker`` function."""
+        return getattr(self.faker.unique, 'bothify')(text='??????')
```

Comment:
why do we need to use getattr here?

Reply:
I just copied over what we had on the other method in order to be the same but you are right there is no need to have the getattr if we are not even treating it.

I removed it here: https://github.com/sdv-dev/RDT/pull/986/commits/34d54931ef9c1bfa9fc3a0589882d8e3287a70a5